stubs/src/radio/radio-controller.ts

```diff
@@ -142,24 +137,26 @@ export class RadioController {
           }
         });
         // Start listening for responses.
-        socket.bind(listenPort);
-        // Send the UDP message, forwarding the given parameters.
-        socket.send(payload, port, address, error => {
-          if (error) {
-            throw new Error(`Failed to send UDP message: ${error.message}`);
-          }
-          console.log('Sent UDP message: ', payload);
-          if (expectedResponsePackets === 0) {
-            // Resolve early if we aren't expecting any response.
-            resolve(new UdpResponse());
-          }
+        socket.bind(listenPort, () => {
```

Comment:
curious what this fix is about?

Reply:
It's to ensure `socket.send()` is called after `socket.bind()`.  If I'm not mistaken, `socket.send()` will still work without the socket being bound, but it will resort to a default unwanted behavior.  I'll see if I can move this into another PR.

Reply:
Done. See #60